packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -150,82 +219,127 @@ class MemoryBodyState extends State<MemoryBody> {
 */
   }
 
+  static const minIncludeTextLeftButtons = 1300;
+  static const minIncludeTextRightButtons = 1100;
+
   Widget _leftsideButtons() {
```

Comment:
nit: rename these methods to something other than left / right. Perhaps _buildPrimaryStateControls for this one and _buildMemoryControls for the other

Reply:
Done.